[2025-02-27T04:29:30.502+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-27T04:29:30.523+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-02-27T04:29:30.537+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-02-27T04:29:30.538+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-27T04:29:30.554+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-26 00:00:00+00:00
[2025-02-27T04:29:30.564+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=17581) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-27T04:29:30.565+0000] {standard_task_runner.py:72} INFO - Started process 17638 to run task
[2025-02-27T04:29:30.566+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-02-26T00:00:00+00:00', '--job-id', '588', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpgvszh8b7']
[2025-02-27T04:29:30.567+0000] {standard_task_runner.py:105} INFO - Job 588: Subtask second_task
[2025-02-27T04:29:30.622+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [running]> on host 25887b48f6fb
[2025-02-27T04:29:30.710+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-26T00:00:00+00:00'
[2025-02-27T04:29:30.711+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-27T04:29:30.712+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-27T04:29:30.712+0000] {logging_mixin.py:190} INFO - Current task name:second_task state:running start_date:2025-02-27 04:29:30.524385+00:00
[2025-02-27T04:29:30.712+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag_v1 and current dag run status:running
[2025-02-27T04:29:30.712+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-27T04:29:30.713+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-27T04:29:30.713+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-02-27T04:29:30.727+0000] {subprocess.py:99} INFO - Output:
[2025-02-27T04:29:30.729+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-02-27T04:29:30.730+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-27T04:29:30.766+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-27T04:29:30.766+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2025-02-26T00:00:00+00:00, execution_date=20250226T000000, start_date=20250227T042930, end_date=20250227T042930
[2025-02-27T04:29:30.794+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-27T04:29:30.794+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-27T04:29:30.795+0000] {logging_mixin.py:190} INFO - Dag name:our_first_dag_v1 queued_at:2025-02-27 04:29:27.210181+00:00
[2025-02-27T04:29:30.795+0000] {logging_mixin.py:190} INFO - Task hostname:25887b48f6fb operator:BashOperator
[2025-02-27T04:29:30.820+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-27T04:29:30.834+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-27T04:29:30.836+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-04T05:11:19.299+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-04T05:11:19.323+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-03-04T05:11:19.333+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-03-04T05:11:19.334+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-04T05:11:19.366+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): second_task> on 2025-02-26 00:00:00+00:00
[2025-03-04T05:11:19.376+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-02-26T00:00:00+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmp95b9r2b3']
[2025-03-04T05:11:19.378+0000] {standard_task_runner.py:105} INFO - Job 14: Subtask second_task
[2025-03-04T05:11:19.380+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=240) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-04T05:11:19.381+0000] {standard_task_runner.py:72} INFO - Started process 253 to run task
[2025-03-04T05:11:19.422+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [running]> on host e52f68431fc8
[2025-03-04T05:11:19.507+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-26T00:00:00+00:00'
[2025-03-04T05:11:19.508+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-04T05:11:19.521+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-03-04T05:11:19.521+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-04T05:11:19.533+0000] {subprocess.py:99} INFO - Output:
[2025-03-04T05:11:19.535+0000] {subprocess.py:106} INFO - hey, I am task2 and will be running after task1!
[2025-03-04T05:11:19.536+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-03-04T05:11:19.565+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-04T05:11:19.566+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, run_id=scheduled__2025-02-26T00:00:00+00:00, execution_date=20250226T000000, start_date=20250304T051119, end_date=20250304T051119
[2025-03-04T05:11:19.636+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-04T05:11:19.653+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-03-04T05:11:19.656+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-03-07T10:44:07.900+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-03-07T10:44:07.907+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [queued]>
[2025-03-07T10:44:07.908+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-03-07T10:44:07.920+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): second_task> on 2025-02-26 00:00:00+00:00
[2025-03-07T10:44:07.926+0000] {standard_task_runner.py:57} INFO - Started process 342 to run task
[2025-03-07T10:44:07.929+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v1', 'second_task', 'scheduled__2025-02-26T00:00:00+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpv203fsh2']
[2025-03-07T10:44:07.931+0000] {standard_task_runner.py:85} INFO - Job 18: Subtask second_task
[2025-03-07T10:44:07.985+0000] {task_command.py:416} INFO - Running <TaskInstance: our_first_dag_v1.second_task scheduled__2025-02-26T00:00:00+00:00 [running]> on host 93934f8e361f
[2025-03-07T10:44:08.066+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v1' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-26T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-26T00:00:00+00:00'
[2025-03-07T10:44:08.067+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-07T10:44:08.068+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo hey, I am task2 and will be running after task1!']
[2025-03-07T10:44:08.083+0000] {subprocess.py:86} INFO - Output:
[2025-03-07T10:44:08.085+0000] {subprocess.py:93} INFO - hey, I am task2 and will be running after task1!
[2025-03-07T10:44:08.085+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-07T10:44:08.106+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v1, task_id=second_task, execution_date=20250226T000000, start_date=20250307T104407, end_date=20250307T104408
[2025-03-07T10:44:08.141+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-03-07T10:44:08.161+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
